data/Sajiyah-Salat.json

```
@@ -1,34 +1,106 @@
 {
   "name": "Sajiyah Salat",
   "type": "personal",
-  "bio": "Open Source Enthusiast!",
+  "bio": "In Search of DevOps Internships | Hacktoberfest 2022 Winner | Technical Writer at Hashnode | Open Source Project Maintainer and Active Contributor to Kubernetes.",
 
-  "tags": ["DevOps", "Docker", "Kubernetes"],
+  "tags": ["DevOps", "Docker", "Kubernetes", "Golang", "Java", "C", "AWS", "Github"],
+"socials": [
+    {
+      "icon": "SiYoutube",
+      "url": "https://www.youtube.com/@sajiyasalat"
+    },
+    {
+      "icon": "FaGithub",
+      "url": "https://github.com/Sajiyah-Salat"
+    },
+    {
+      "icon": "FaPencilAlt",
+      "url": "https://medium.com/@Sajiyah-Salat"
+    },
+    {
+      "icon": "FaTwitter",
+      "url": "https://twitter.com/SajiyaSalat"
+    },
+     {
+      "icon": "FaLinkedin",
+      "url": "https://www.linkedin.com/in/sajiya-salat-0a2a78245/"
+    }
+  ],
   "links": [
     {
-      "name": "Follow me on GitHub",
+      "name": "Checkout my work",
       "url": "https://github.com/Sajiyah-Salat",
       "icon": "FaGithub"
     },
     {
-      "name": "Follow me on LinkedIN",
+      "name": "Know my tech journey",
       "url": "https://www.linkedin.com/in/sajiya-salat-0a2a78245/",
       "icon": "FaLinkedin"
     },
     {
-      "name": "Follow me on Twitter",
+      "name": "Tips, tricks, hacks",
       "url": "https://twitter.com/SajiyaSalat",
       "icon": "FaTwitter"
+    },
+     {
+      "name": "Read my Blogs",
+      "url": "https://medium.com/@Sajiyah-Salat",
+      "icon": "FaMedium"
+    },
+        {
+      "name": "Checkout my videos",
+      "url": "https://www.youtube.com/@sajiyasalat",
+      "icon": "FaYoutube"
     }
-  ],
+    ],
   "milestones": [
-    {
+  {
       "title": "Won Hacktoberfest 2022 ",
       "date": "October 2022",
-      "icon": "hacktoberfest",
-      "color": "grey",
-      "description": "Made Github profile on June 2022 | Started documenting my journey via linkedIn and Twitter| Started Writing blog on Hashnode",
-      "url": "https://www.Sajiyah-Salat.io/"
+      "icon": "SiGithub",
+      "color": "black",
+      "description": "Made Github profile on June 2022 | Started Contributing in Multiple Projects.",
+      "url": "https://github.com/Sajiyah-Salat"
+    },
+      {
+      "title": "Started Documenting my journey on twitter and linkedin",
+      "date": "June 2022",
+      "icon": "SiLinkedin",
+      "color": "blue",
+      "description": "I post daily about my learnings on twitter and linkedin. Many poeple support my work and I have made great connections.",
+      "url": "https://www.linkedin.com/in/sajiya-salat-0a2a78245/"
+    },
+      {
+      "title": "Started Writitng Blogs",
```

you can fix this typo in title
```suggestion
      "title": "Started Writing Blogs",
```